### PR TITLE
remove regexes in isLoopback, use isLoopback in isPrivate, write less in toBuffer, add describe in api-test.js

### PR DESCRIPTION
**Pull request by blitmap**
Hello - I was using this module for something and while looking through the source I discovered a few oddities?

I thought I'd contribute :-)

### CODE REVIEW

File: test/api-test.js [hunk 1]
```diff
@@ -1,7 +1,8 @@
 var ip = require('..'),
     assert = require('assert'),
     net = require('net'),
-    os = require('os');
+    os = require('os'),
+    describe = require('describe');
```
Comment (indutny) [inline test/api-test.js]: This is irrelevant.

**Comment by indutny** [comment]
Thank you for posting this! May I ask you to rebase your PR over the current master, and fix the lint issues?

Cheers!

**Comment by blitmap** [comment]
Sure thing - didn't realize I typo'd adding decribe in api-test, but I can leave that out :-)

**Comment by blitmap** [comment]
I accidentally created a separate pull request.. so er.. closing this one. :>

**Pull request closed without merging**